system/http.go

```diff
@@ -86,6 +88,14 @@ func (u *DefHTTP) setup() error {
 		return u.err
 	}
 	req.Header = u.RequestHeader.Clone()
+
+	if host, ok := u.RequestHeader["Host"]; ok {
+		if len(host) > 1 {
+			fmt.Fprintf(os.Stderr, "multiple host headers detected, using first: (%v)", host[0])
```

Comment:
these should not produce output like this, goss can be used as a package by other projects and this is not a helpful way to communicate back this information in that way - there's no way to do this in current goss.

Reply:
👍 will remove